src/analysis/Environments.test.ts

```diff
@@ -69,6 +69,11 @@ jest.mock('src/libs/state', () => ({
   getUser: jest.fn(),
 }));
 
+jest.mock('src/components/Modal', () => {
```

Comment:
This was failing in ci (but testing locally). @msilva-broad had seen this type of issue in the past and recommended using the mock modal component, which seems to have fixed the flakiness.

Reply:
Note, this change was added further up in the file on dev. So this change does not occur on React 18 rebase branch.